diff --git a/dist/node/node.d.ts b/dist/node/node.d.ts
index 0b5368a840d8b6b9e22a76e98ec77e02325cac6c..bac6740335116832d42478124337a2e198a70cda 100644
--- a/dist/node/node.d.ts
+++ b/dist/node/node.d.ts
@@ -1824,7 +1824,7 @@ declare namespace ContentDelta {
     }
     namespace FunctionResultDelta {
         interface Items {
-            items?: Array<string | InteractionsAPI.ImageContent | unknown>;
+            items?: Array<string | InteractionsAPI.ImageContent>;
         }
     }
     interface CodeExecutionCallDelta {
@@ -1917,7 +1917,7 @@ declare namespace ContentDelta {
     }
     namespace MCPServerToolResultDelta {
         interface Items {
-            items?: Array<string | InteractionsAPI.ImageContent | unknown>;
+            items?: Array<string | InteractionsAPI.ImageContent>;
         }
     }
     interface FileSearchCallDelta {
@@ -3892,7 +3892,7 @@ declare interface FunctionResultContent {
     /**
      * The result of the tool call.
      */
-    result: FunctionResultContent.Items | unknown | string;
+    result: FunctionResultContent.Items | string;
     type: 'function_result';
     /**
      * Whether the tool call resulted in an error.
@@ -3906,7 +3906,7 @@ declare interface FunctionResultContent {
 
 declare namespace FunctionResultContent {
     interface Items {
-        items?: Array<string | InteractionsAPI.ImageContent | unknown>;
+        items?: Array<string | InteractionsAPI.ImageContent>;
     }
 }
 
@@ -7048,7 +7048,7 @@ declare interface MCPServerToolResultContent {
     /**
      * The result of the tool call.
      */
-    result: MCPServerToolResultContent.Items | unknown | string;
+    result: MCPServerToolResultContent.Items | string;
     type: 'mcp_server_tool_result';
     /**
      * Name of the tool which is called for this specific tool call.
@@ -7062,7 +7062,7 @@ declare interface MCPServerToolResultContent {
 
 declare namespace MCPServerToolResultContent {
     interface Items {
-        items?: Array<string | InteractionsAPI.ImageContent | unknown>;
+        items?: Array<string | InteractionsAPI.ImageContent>;
     }
 }
 
@@ -7130,7 +7130,7 @@ export declare enum MediaResolution {
  * This type contains `RequestInit` options that may be available on the current runtime,
  * including per-platform extensions like `dispatcher`, `agent`, `client`, etc.
  */
-declare type MergedRequestInit = RequestInits & 
+declare type MergedRequestInit = RequestInits &
 /** We don't include these in the types as they'll be overridden for every request. */
 Partial<Record<'body' | 'headers' | 'method' | 'signal', never>>;
 
@@ -8619,7 +8619,7 @@ export declare interface Schema {
     type?: Type;
 }
 
-export declare type SchemaUnion = Schema | unknown;
+export declare type SchemaUnion = Schema;
 
 /** An image mask representing a brush scribble. */
 export declare interface ScribbleImage {
