diff --git a/src/resources/responses/responses.ts b/src/resources/responses/responses.ts
index 8542619e5671278dcf74ff323bd6b9755d144c3b..b5d10ef6431c9aaabdc39baba79f4450d35bed2d 100644
--- a/src/resources/responses/responses.ts
+++ b/src/resources/responses/responses.ts
@@ -3523,12 +3523,20 @@ export namespace ResponseOutputItem {
      */
     status: 'in_progress' | 'completed' | 'generating' | 'failed';
 
+    background: "opaque" | "transparent" | "auto";
+
+    output_format: "png" | "jpeg" | "webp";
+    quality: "high" | "medium" | "low" | "auto";
+
+    revised_prompt: string | null;
+
     /**
      * The type of the image generation call. Always `image_generation_call`.
      */
     type: 'image_generation_call';
   }
 
+
   /**
    * A tool call to run a command on the local shell.
    */
